tests/queries/0_stateless/03604_test_merge_tree_min_read_task_size_is_zero.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Btw, `.sql` tests are preferred over `.sh` tests whenever possible. There doesn't seem to be any particular reason that it should be a `.sh` test - or is there one?